kcidb/monitor/output.py

```diff
@@ -136,9 +138,25 @@ def render(self):
             ready to send, but missing the From header.
         """
         email = EmailMessage()
-        email["Subject"] = self.message.summary + self.obj.summarize()
+        subject = self.message.summary + self.obj.summarize()
+        email["Subject"] = subject
         email["To"] = ", ".join(self.message.recipients)
         email["X-KCIDB-Notification-ID"] = self.id
         email["X-KCIDB-Notification-Message-ID"] = self.message.id
-        email.set_content(self.message.description + self.obj.describe())
+        message = self.message.description + self.obj.describe()
+        email.set_content(message)
+        escaped_subject = html.escape(subject, quote=True)
+        escaped_message = html.escape(message, quote=True)
+        escaped_message = re.sub(r'((http|https|git|ftp)://[^\s]+)', escaped_message)
```

Comment:
Thanks! The regular expression looks good to me now. However, this invocation of re.sub() won't work correctly. Check the [usage docs](https://docs.python.org/3/library/re.html#re.sub).

Also you can make pylint happy if you wrap this line.

Reply:
Arh.... I forgot to put back the repl. Will do so now

Reply:
@spbnick, I made changes. Please review